cypress/integration/workspace.test.ts

```diff
@@ -5,8 +5,8 @@ context("Test the overall app", () => {
 
   describe("Desktop functionalities", () => {
     it("renders with text", () => {
-      cy.get('.app-container').should('exist');
-      cy.get('.nav-and-content-container').should('exist');
+      cy.get(".app-container").should("exist");
+      cy.get(".nav-and-content-container").should("exist");
```

Comment:
Thanks for fixing this!  At one point, I was tinkering with Cypress syntax to see if it accepted both single and double quotes - must have overlooked restoring it.  I'm actually not a fan of single quotes unless they are absolutely needed.